package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "httpntlm",
+  "name": "@r24y/httpntlm",
```

Comment:
I'm so sorry, I pushed my personal fork to npm under my username and didn't realize I'd kept the change in the branch I pushed for this PR. I'll correct this and push again.